predeployed/setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = oracle-predeployed
+version = file: version.txt
+author = Artem Payvin
+author_email = artem.payvin@skalelabs.com
+description = A tool for generating predeployed oracle smart contract
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/skalenetwork/oracle-contract
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU Affero General Public License v3
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+include_package_data = True
+python_requires = >=3.7
+install_requires =
+    predeployed-generator >= 0.0.1a4
```

Comment:
Too old version of `predeployed-generator`.

Reply:
Agree will update